src/luarocks/cmd/install.lua

```diff
@@ -42,7 +42,7 @@ function install.add_to_parser(parser)
    util.deps_mode_option(cmd)
    -- luarocks build options
    parser:flag("--pack-binary-rock"):hidden(true)
-   parser:flag("--branch"):hidden(true)
+   parser:option("--branch"):argname("<name>"):hidden(true)
```

Comment:
The argname doesn't matter since it's hidden.

Reply:
:+1: !